core/meta/inc/TClass.h

```diff
@@ -309,12 +311,14 @@ friend class TProtoClass;
    TClass(const TClass& tc) = delete;
    TClass& operator=(const TClass&) = delete;
 
+   void LoadClassInfo() const;
+
 protected:
-   TVirtualStreamerInfo     *FindStreamerInfo(TObjArray* arr, UInt_t checksum) const;
-   void                      GetMissingDictionariesForBaseClasses(TCollection& result, TCollection& visited, bool recurse);
-   void                      GetMissingDictionariesForMembers(TCollection& result, TCollection& visited, bool recurse);
-   void                      GetMissingDictionariesWithRecursionCheck(TCollection& result, TCollection& visited, bool recurse);
-   void                      GetMissingDictionariesForPairElements(TCollection& result, TCollection& visited, bool recurse);
+   TVirtualStreamerInfo *FindStreamerInfo(TObjArray* arr, UInt_t checksum) const;
```

Comment:
The extra space is kinda ugly, and violates the coding conventions by making the lines too long. In any case, I will probably just drop the formatting commit and leave it for another time.